pyramus/src/main/webapp/scripts/ixtable/ixtable.js

```diff
@@ -1909,8 +1909,24 @@ IxButtonTableEditorButtonController = Class.create(IxTableEditorController, {
       cellViewer._columnDefinition = columnDefinition;
       
       return cellViewer;
-    } else {
-      throw new Error("Unable to build button without image");
+    }
+    else if (columnDefinition.value) {
```

Comment:
This is technically not needed for this feature but since IxTable only supported image buttons, I ended up making it support ordinary buttons with text as well.